conformance/fhir-ig-us-core/src/test/java/com/ibm/fhir/ig/us/core/tool/ResourceProcessor.java

```diff
@@ -36,8 +43,33 @@ public static void main(String[] args) throws Exception {
             throw new IllegalArgumentException("Please specify one or more version qualifiers (e.g. '4.0.0 5.0.0')");
         }
 
-        for (String packageVersion : args) {
-            updateInPlace(packageVersion.replace(".", ""));
+        for (String version : args) {
+            String packageVersion = version.replace(".", "");
```

Comment:
will this ever end up with unintended collisions? I guess as long as there are always 3 numbers in version, it should be OK

Reply:
yeah, its just a tool we use to process the conformance artifacts and so I thought the heuristic would be ok here